### PR TITLE
MEX-163 HTN dash should have Cholesterol & BMI graphs

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/MEX-163

### CODE REVIEW

**Comment by brandones** [comment]
This is a minor refactor of CALF that introduces a GraphFactory.

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -92,6 +91,19 @@
 
     private String patientVisitsPageWithSpecificVisitUrl = "";
 
+    @Autowired
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java]: https://stackoverflow.com/questions/40620000/spring-autowire-on-properties-vs-constructor

File: api/src/main/resources/messages.properties [hunk 2]
```diff
@@ -663,12 +663,6 @@ providermanagement.assignPatientToProvider=Assign Patient to Provider
 
 pih.app.cohortBuilder.label=Cohort Builder
 
-pih.app.bloodPressure.obsTable.title=Blood Pressure Observations
```
Comment (brandones) [inline api/src/main/resources/messages.properties]: There's a couple new codes and a couple removed, but mostly this a reorder and alphebetization.

**Comment by mogoodrich** [comment]
LGTM... thanks for reorganizing/pulling this out into reusable chunks...

**Comment by brandones** [comment]
Thanks Mark!

**Pull request merged**